pipeline/bibframesource.py

```diff
@@ -818,11 +814,8 @@ def is_swedishlist(self):
     @property
     def ssif_1_codes(self):
         uka_subject_codes = []
-        for subject in self.bibframe_master.get("instanceOf", {}).get("subject", []):
-            if (
-                subject.get("inScheme", {}).get("code", "") == "uka.se"
-                and subject.get("@type", "") == "Topic"
-            ):
+        for subject in self.bibframe_master.get("instanceOf", {}).get("classification", []):
+            if is_ssif_classification(term):
```

Comment:
```suggestion
            if is_ssif_classification(subject):
```
`subject` vs `term` (`term` was undefined)